tests/logictest/suites/ydb/select1-1.test

```diff
@@ -228,35 +228,35 @@ where d not between 110 and 150
 order by col1,col5,col3,col2,col4;
 
 ----
-102 2 111 1014 103
```

Comment:
Does this mean the original result was wrong? 🤣 

---

Let's find a better way to check results, not manually. @ZeaLoVe 

There are several bad points in this current approach:
- We need to compare with other databases manually, making mistakes easy to make, such as the case.
- We need to manually write answers which is inefficient, though we can copy answers from database's output, we also need to process format.

Reply:
The result is generate by query databend through http handler, it may be wrong after feature changed. maybe use another database is good idea.  Genrator of test suites now has a issue #6709 .